sklearn/tree/tests/test_tree.py

```diff
@@ -1325,3 +1335,36 @@ def test_presort_sparse():
 
     for est, sparse_matrix in product(ests, sparse_matrices):
         yield check_presort_sparse, est, sparse_matrix(X), y
+
+
+def check_decision_path(name):
+    X = iris.data
```

Comment:
Is it possible to include a small example with a hardcoded decision path? That'll ensure we're getting the right result. Maybe just a training set with 4-5 points, and a testing set of 2 points.